results.csv

```diff
@@ -0,0 +1,15 @@
+sim_count,starting_bet,result_sum
+100,1,-9.5
+100,1,-5.0
+100,1,3.0
```

Comment:
Are these result sums looking like what we expect? I was surprised to see them flipping from negative to positive.

Reply:
I think that's expected with that number of trials.
We will probably need at least 10 thousand trials for the numbers to become more like we expected.

@esulu can provide more info on this since he did most of the testing with the Game class and its returns.

Reply:
They sometimes flip even with ~100k trials. When I tested with 1 million games they seem to consistently be negative. 

Nabeel and I were discussing if the PRNG used in the library card drawing would have an effect at all, and I'm still not completely sure

Reply:
Cool, I increased it to 1000000 for now.